src/pages/user/User.js

```diff
@@ -22,22 +24,34 @@ class User extends Component {
     })
   }
 
-  async componentWillMount() {
-    this.props.fetchUser(
-      this.props.userAddress,
-      this.props.intl.formatMessage(this.intlMessages.unnamedUser)
-    )
-    // TODO: User reviews
+  async componentDidMount() {
+    const { fetchUser, intl, userAddress } = this.props
+
+    fetchUser(userAddress, intl.formatMessage(this.intlMessages.unnamedUser))
+
+    const listingIdsAsSeller = await origin.marketplace.getListings({
```

Comment:
Why fetching those seller side reviews since we then filter them on line #54 ?

Reply:
@franckc unless I misunderstood how this is working, we need to fetch all the reviews from listings where the user was buyer or seller. For each listing, that will return up to two reviews. Then we need to filter them for basically every other review. In other words, just because the user was a seller on a particular listing doesn’t mean that the review will get filtered out. 🧐

Reply:
Yes, you make sense and I don't !